daemon/graphdriver/driver.go

```diff
@@ -85,6 +86,26 @@ type Driver interface {
 	DiffSize(id, parent string) (size int64, err error)
 }
 
+// CustomImageAccessor is an interface to allow callers to query the object for
+// any implementation specific image information that might need to be inserted into
+// the graph.
+type CustomImageAccessor interface {
```

Comment:
Let's add a comment here that this interface is for windows support. Warn any users that this is subject to change.